### DISCUSSION TITLE
嘉立创EDA 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#148] [模板讨论]
嘉立创EDA 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by as932256** [comment]
大佬，嘉立创签到的时候在第五步报错
<html>
<body>


Failed at 5/11 request,Fail assert: {"re": "302", "from": "status"} from success_asserts,Response Error : HTTP 599: Connection timed out after 30001 milliseconds,Request URL: https://passport.szlcsc.com/login
--



</body>
</html>

**Reply by wjf0214** [reply]
更新了，大概看了下，账号登录加了验证。 我改成cookie模式，你试试吧。

**Comment by goxofy** [comment]
```
2023-07-13 13:17:57
日志：Failed at 1/4 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 302: Found,
Request URL: https://oshwhub.com/sign_in
```

大佬签到提示这个错误，但是手动更新 cookie 之后就能成功，但是第二天又会提示这个

**Reply by wjf0214** [reply]
cookie现在有效期很短，第二天就过期了。所以失败了。然后登陆加了有验证，目前也不太好做账密登录的方式。

**Reply by goxofy** [reply]
soga 明白了

**Reply by Snitro** [reply]
现在是每天都需要添加cookie才能签到是吗？感觉有点麻烦呢，以后是否还会再维护呢？

**Reply by wjf0214** [reply]
目前来说是这样的。早期的版本是账密登录，不用担心失效，但是后面加了滑块验证登加密方式，没办法，就只能是cookie方式了，但是cookie每天都失效。目前也没特别好的方式。

**Reply by hyjiangqing** [reply]
同样发现隔一天后会失效
